schemas/veres-one-validator.js

```diff
@@ -20,12 +27,18 @@ const caveat = {
   properties: {
     type: {
       type: 'string',
+      maxLength
       // FIXME: enable when term is finalized
       // enum: ['VeresOneElectorTicketAgent']
     }
   }
 };
 
+const invocationTarget = {
+  // FIXME this will allow any string with minLength 1 or more
+  anyOf: [schemas.url(), did(), didReference(), didUuid()]
```

Comment:
Why isn't this more restrictive?

Reply:
`invocationTarget` can be a `uuid`, `did` or `didUuid` now: https://github.com/veres-one/veres-one-validator/pull/38/commits/18dc092996069267c1b807001b1aa31f43402744